clang/lib/Driver/ToolChains/SYCL.cpp

```diff
@@ -1780,6 +1780,13 @@ void SYCLToolChain::AddImpliedTargetArgs(const llvm::Triple &Triple,
     ArgStringList TargArgs;
     Args.AddAllArgValues(TargArgs, options::OPT_Xs, options::OPT_Xs_separate);
     Args.AddAllArgValues(TargArgs, options::OPT_Xsycl_backend);
+    // For -Xsycl-target-backend=<triple> we need to scrutinize the triple
```

Comment:
It is not clear to me what this comment means?
Can you help add some more description?

Reply:
sure - I'll update the comment